skimage/transform/setup.py

```diff
@@ -30,12 +34,13 @@ def configuration(parent_package='', top_path=None):
     return config
 
 if __name__ == '__main__':
-    from numpy.distutils.core import setup
+    from numpy.distutils.core import setup, Extension
     setup(maintainer='scikit-image Developers',
           author='scikit-image Developers',
           maintainer_email='scikit-image@googlegroups.com',
           description='Transforms',
           url='https://github.com/scikit-image/scikit-image',
           license='SciPy License (BSD Style)',
+          ext_modules=[Extension(extra_compile_args=['/EHsc']),],
```

Comment:
That line should not be here. I was trying something to fix the round problem and I didn't correctly kick out the commit.